### PR TITLE
Barebone Test for Travis CI

### PR DESCRIPTION
**Pull request by boggy-d**
## Description

Travis CI needs tests to pass in order to work. Combining it with a always-passing Jest test makes testing the Travis CI build possible

## Changes proposed

 - Change 1
 - Change 2
 - etc..

## How to test it

 - Run specs/tests

## Next steps

None.

## Deploy notes

None.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,5 +8,6 @@ cache: npm
 
 script:
   - cd neutralino.js
+  - npm install
   - npm test
-  
+
```
Comment (AndreiUngur) [inline .travis.yml:13]: nit: terminate file with newline

**Review by AndreiUngur — APPROVED** [review]
LGTM!
Small nitpick comment other than the one about newlines: the changes proposed are pretty lightweight but the commit history is not, since you've experimented with Travis' builds in the process. It'd be great if you could squash all these commits into a single one.

I'm approving despite the nitpicky comments because [it is definitely functional](https://travis-ci.com/scrum-gang/neutralinojs/jobs/258316287) and, given the nature of this repository, those issues are no big deal to me.

**Review by kraglalbert — APPROVED** [review]
LGTM!

File: .travis.yml (hunk 1 above)
Comment (boggy-d) [reply .travis.yml:13]: isn't that what's up here?

**Comment by boggy-d** [comment]
>  It'd be great if you could squash all these commits into a single one.

I 100% agree, I'm gonna squash during the merge

**Pull request merged**